### ISSUE TITLE
Option 'report-dir' cannot be cast to type 'org.gradle.api.file.Directory' in class 'io.qameta.allure.gradle.report.tasks.AllureReport'.

### ISSUE THREAD

**Issue by donbeave** [allure-framework/allure-gradle#69]
#### I'm submitting a ... 
  - [x] bug report

#### What is the current behavior?

I'm getting this error by using the latest Allure Gradle plugin:

```
~/Projects/myorg/my-project master ❯ ./gradlew allureReport --depends-on-tests --stacktrace --info                                                                        1m 51s
Initialized native services in: /Users/donbeave/.gradle/native
Initialized jansi services in: /Users/donbeave/.gradle/native
The client will now receive all logging from the daemon (pid: 22009). The daemon log file: /Users/donbeave/.gradle/daemon/7.2/daemon-22009.out.log
Starting 2nd build in daemon [uptime: 2 mins 5.023 secs, performance: 100%]
Using 12 worker leases.
Now considering [/Users/donbeave/Projects/myorg/my-project, /Users/donbeave/Projects/myorg/my-project/buildSrc] as hierarchies to watch
Watching the file system is configured to be enabled if available
File system watching is active
Starting Build
Settings evaluated using settings file '/Users/donbeave/Projects/myorg/my-project/settings.gradle.kts'.
Projects loaded. Root project using build file '/Users/donbeave/Projects/myorg/my-project/build.gradle.kts'.
Included projects: [root project 'my-project']
Now considering [/Users/donbeave/Projects/myorg/my-project/buildSrc, /Users/donbeave/Projects/myorg/my-project] as hierarchies to watch

> Configure project :buildSrc
Evaluating project ':buildSrc' using build file '/Users/donbeave/Projects/myorg/my-project/buildSrc/build.gradle.kts'.
kotlin scripting plugin: created the scripting discovery configuration: kotlinScriptDef
kotlin scripting plugin: created the scripting discovery configuration: testKotlinScriptDef
file or directory '/Users/donbeave/Projects/myorg/my-project/buildSrc/src/main/resources', not found
Caching disabled for Kotlin DSL accessors for project ':buildSrc' because:
  Build cache is disabled
Skipping Kotlin DSL accessors for project ':buildSrc' as it is up-to-date.
file or directory '/Users/donbeave/Projects/myorg/my-project/buildSrc/src/main/resources', not found
Selected primary task 'build' from project :
file or directory '/Users/donbeave/Projects/myorg/my-project/buildSrc/src/main/resources', not found
file or directory '/Users/donbeave/Projects/myorg/my-project/buildSrc/src/main/resources', not found
:buildSrc:extractPrecompiledScriptPluginPlugins (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:extractPrecompiledScriptPluginPlugins UP-TO-DATE
Caching disabled for task ':buildSrc:extractPrecompiledScriptPluginPlugins' because:
  Build cache is disabled
Skipping task ':buildSrc:extractPrecompiledScriptPluginPlugins' as it is up-to-date.
:buildSrc:extractPrecompiledScriptPluginPlugins (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.004 secs.
:buildSrc:generateExternalPluginSpecBuilders (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:generateExternalPluginSpecBuilders UP-TO-DATE
Caching disabled for task ':buildSrc:generateExternalPluginSpecBuilders' because:
  Build cache is disabled
Skipping task ':buildSrc:generateExternalPluginSpecBuilders' as it is up-to-date.
:buildSrc:generateExternalPluginSpecBuilders (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.007 secs.
:buildSrc:compilePluginsBlocks (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:compilePluginsBlocks UP-TO-DATE
Caching disabled for task ':buildSrc:compilePluginsBlocks' because:
  Build cache is disabled
Skipping task ':buildSrc:compilePluginsBlocks' as it is up-to-date.
:buildSrc:compilePluginsBlocks (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.003 secs.
:buildSrc:generatePrecompiledScriptPluginAccessors (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:generatePrecompiledScriptPluginAccessors UP-TO-DATE
Caching disabled for task ':buildSrc:generatePrecompiledScriptPluginAccessors' because:
  Build cache is disabled
Skipping task ':buildSrc:generatePrecompiledScriptPluginAccessors' as it is up-to-date.
:buildSrc:generatePrecompiledScriptPluginAccessors (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.009 secs.
:buildSrc:configurePrecompiledScriptDependenciesResolver (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:configurePrecompiledScriptDependenciesResolver
Caching disabled for task ':buildSrc:configurePrecompiledScriptDependenciesResolver' because:
  Build cache is disabled
Task ':buildSrc:configurePrecompiledScriptDependenciesResolver' is not up-to-date because:
  Task has not declared any outputs despite executing actions.
:buildSrc:configurePrecompiledScriptDependenciesResolver (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.004 secs.
:buildSrc:extractPluginRequests (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:extractPluginRequests UP-TO-DATE
Caching disabled for task ':buildSrc:extractPluginRequests' because:
  Build cache is disabled
Skipping task ':buildSrc:extractPluginRequests' as it is up-to-date.
:buildSrc:extractPluginRequests (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.0 secs.
:buildSrc:generatePluginAdapters (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:generatePluginAdapters UP-TO-DATE
Caching disabled for task ':buildSrc:generatePluginAdapters' because:
  Build cache is disabled
Skipping task ':buildSrc:generatePluginAdapters' as it is up-to-date.
:buildSrc:generatePluginAdapters (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.0 secs.
:buildSrc:generateScriptPluginAdapters (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:generateScriptPluginAdapters UP-TO-DATE
Caching disabled for task ':buildSrc:generateScriptPluginAdapters' because:
  Build cache is disabled
Skipping task ':buildSrc:generateScriptPluginAdapters' as it is up-to-date.
:buildSrc:generateScriptPluginAdapters (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.0 secs.
:buildSrc:compileKotlin (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:compileKotlin UP-TO-DATE
Caching disabled for task ':buildSrc:compileKotlin' because:
  Build cache is disabled
Skipping task ':buildSrc:compileKotlin' as it is up-to-date.
:buildSrc:compileKotlin (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.042 secs.
:buildSrc:compileJava (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:compileJava UP-TO-DATE
Caching disabled for task ':buildSrc:compileJava' because:
  Build cache is disabled
Skipping task ':buildSrc:compileJava' as it is up-to-date.
:buildSrc:compileJava (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.008 secs.
:buildSrc:compileGroovy (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:compileGroovy NO-SOURCE
Skipping task ':buildSrc:compileGroovy' as it has no source files and no previous output files.
:buildSrc:compileGroovy (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.001 secs.
:buildSrc:compileGroovyPlugins (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:compileGroovyPlugins UP-TO-DATE
Caching disabled for task ':buildSrc:compileGroovyPlugins' because:
  Build cache is disabled
Skipping task ':buildSrc:compileGroovyPlugins' as it is up-to-date.
:buildSrc:compileGroovyPlugins (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.003 secs.
:buildSrc:pluginDescriptors (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:pluginDescriptors UP-TO-DATE
Caching disabled for task ':buildSrc:pluginDescriptors' because:
  Build cache is disabled
Skipping task ':buildSrc:pluginDescriptors' as it is up-to-date.
:buildSrc:pluginDescriptors (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.001 secs.
:buildSrc:processResources (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:processResources UP-TO-DATE
file or directory '/Users/donbeave/Projects/myorg/my-project/buildSrc/src/main/resources', not found
Caching disabled for task ':buildSrc:processResources' because:
  Build cache is disabled
Skipping task ':buildSrc:processResources' as it is up-to-date.
:buildSrc:processResources (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.002 secs.
:buildSrc:classes (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:classes UP-TO-DATE
Skipping task ':buildSrc:classes' as it has no actions.
:buildSrc:classes (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.0 secs.
:buildSrc:inspectClassesForKotlinIC (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:inspectClassesForKotlinIC UP-TO-DATE
Caching disabled for task ':buildSrc:inspectClassesForKotlinIC' because:
  Build cache is disabled
Skipping task ':buildSrc:inspectClassesForKotlinIC' as it is up-to-date.
:buildSrc:inspectClassesForKotlinIC (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.002 secs.
:buildSrc:jar (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:jar UP-TO-DATE
Caching disabled for task ':buildSrc:jar' because:
  Build cache is disabled
Skipping task ':buildSrc:jar' as it is up-to-date.
:buildSrc:jar (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.003 secs.
:buildSrc:assemble (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:assemble UP-TO-DATE
Skipping task ':buildSrc:assemble' as it has no actions.
:buildSrc:assemble (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.0 secs.
:buildSrc:compileTestKotlin (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:compileTestKotlin NO-SOURCE
file or directory '/Users/donbeave/Projects/myorg/my-project/buildSrc/src/test/kotlin', not found
file or directory '/Users/donbeave/Projects/myorg/my-project/buildSrc/src/test/java', not found
Skipping task ':buildSrc:compileTestKotlin' as it has no source files and no previous output files.
:buildSrc:compileTestKotlin (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.0 secs.
:buildSrc:pluginUnderTestMetadata (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:pluginUnderTestMetadata UP-TO-DATE
Caching disabled for task ':buildSrc:pluginUnderTestMetadata' because:
  Build cache is disabled
Skipping task ':buildSrc:pluginUnderTestMetadata' as it is up-to-date.
:buildSrc:pluginUnderTestMetadata (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.001 secs.
:buildSrc:compileTestJava (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:compileTestJava NO-SOURCE
file or directory '/Users/donbeave/Projects/myorg/my-project/buildSrc/src/test/java', not found
Skipping task ':buildSrc:compileTestJava' as it has no source files and no previous output files.
:buildSrc:compileTestJava (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.0 secs.
:buildSrc:compileTestGroovy (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:compileTestGroovy NO-SOURCE
file or directory '/Users/donbeave/Projects/myorg/my-project/buildSrc/src/test/groovy', not found
Skipping task ':buildSrc:compileTestGroovy' as it has no source files and no previous output files.
:buildSrc:compileTestGroovy (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.0 secs.
:buildSrc:processTestResources (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:processTestResources NO-SOURCE
file or directory '/Users/donbeave/Projects/myorg/my-project/buildSrc/src/test/resources', not found
Skipping task ':buildSrc:processTestResources' as it has no source files and no previous output files.
:buildSrc:processTestResources (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.0 secs.
:buildSrc:testClasses (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:testClasses UP-TO-DATE
Skipping task ':buildSrc:testClasses' as it has no actions.
:buildSrc:testClasses (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.0 secs.
:buildSrc:test (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:test NO-SOURCE
file or directory '/Users/donbeave/Projects/myorg/my-project/buildSrc/build/classes/java/test', not found
file or directory '/Users/donbeave/Projects/myorg/my-project/buildSrc/build/classes/groovy/test', not found
file or directory '/Users/donbeave/Projects/myorg/my-project/buildSrc/build/classes/kotlin/test', not found
Skipping task ':buildSrc:test' as it has no source files and no previous output files.
:buildSrc:test (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.0 secs.
:buildSrc:validatePlugins (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:validatePlugins UP-TO-DATE
Caching disabled for task ':buildSrc:validatePlugins' because:
  Build cache is disabled
Skipping task ':buildSrc:validatePlugins' as it is up-to-date.
:buildSrc:validatePlugins (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.003 secs.
:buildSrc:check (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:check UP-TO-DATE
Skipping task ':buildSrc:check' as it has no actions.
:buildSrc:check (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.0 secs.
:buildSrc:build (Thread[Execution worker for ':buildSrc',5,main]) started.

> Task :buildSrc:build UP-TO-DATE
Skipping task ':buildSrc:build' as it has no actions.
:buildSrc:build (Thread[Execution worker for ':buildSrc',5,main]) completed. Took 0.0 secs.

> Configure project :
Evaluating root project 'my-project' using build file '/Users/donbeave/Projects/myorg/my-project/build.gradle.kts'.
Caching disabled for Kotlin DSL accessors for root project 'my-project' because:
  Build cache is disabled
Skipping Kotlin DSL accessors for root project 'my-project' as it is up-to-date.
All projects evaluated.

FAILURE: Build failed with an exception.

* What went wrong:
Option 'report-dir' cannot be cast to type 'org.gradle.api.file.Directory' in class 'io.qameta.allure.gradle.report.tasks.AllureReport'.

* Try:
Run with --debug option to get more log output. Run with --scan to get full insights.

* Exception is:
org.gradle.api.internal.tasks.options.OptionValidationException: Option 'report-dir' cannot be cast to type 'org.gradle.api.file.Directory' in class 'io.qameta.allure.gradle.report.tasks.AllureReport'.
        at org.gradle.api.internal.tasks.options.AbstractOptionElement.createNotationParserOrFail(AbstractOptionElement.java:86)
        at org.gradle.api.internal.tasks.options.SingleValueOptionElement.<init>(SingleValueOptionElement.java:37)
        at org.gradle.api.internal.tasks.options.AbstractOptionElement.of(AbstractOptionElement.java:56)
        at org.gradle.api.internal.tasks.options.FieldOptionElement.create(FieldOptionElement.java:38)
        at org.gradle.api.internal.tasks.options.OptionReader.getFieldAnnotations(OptionReader.java:110)
        at org.gradle.api.internal.tasks.options.OptionReader.getOptionElements(OptionReader.java:99)
        at org.gradle.api.internal.tasks.options.OptionReader.loadClassDescriptorInCache(OptionReader.java:56)
        at org.gradle.api.internal.tasks.options.OptionReader.getOptions(OptionReader.java:46)
        at org.gradle.execution.commandline.CommandLineTaskConfigurer.configureTasksNow(CommandLineTaskConfigurer.java:51)
        at org.gradle.execution.commandline.CommandLineTaskConfigurer.configureTasks(CommandLineTaskConfigurer.java:44)
        at org.gradle.execution.commandline.CommandLineTaskParser.parseTasks(CommandLineTaskParser.java:45)
        at org.gradle.execution.TaskNameResolvingBuildConfigurationAction.configure(TaskNameResolvingBuildConfigurationAction.java:46)
        at org.gradle.execution.DefaultBuildConfigurationActionExecuter.configure(DefaultBuildConfigurationActionExecuter.java:51)
        at org.gradle.execution.DefaultBuildConfigurationActionExecuter.access$000(DefaultBuildConfigurationActionExecuter.java:25)
        at org.gradle.execution.DefaultBuildConfigurationActionExecuter$1.proceed(DefaultBuildConfigurationActionExecuter.java:59)
        at org.gradle.execution.DefaultTasksBuildExecutionAction.configure(DefaultTasksBuildExecutionAction.java:45)
        at org.gradle.execution.DefaultBuildConfigurationActionExecuter.configure(DefaultBuildConfigurationActionExecuter.java:51)
        at org.gradle.execution.DefaultBuildConfigurationActionExecuter.lambda$select$0(DefaultBuildConfigurationActionExecuter.java:38)
        at org.gradle.internal.Factories$1.create(Factories.java:31)
        at org.gradle.api.internal.project.DefaultProjectStateRegistry.withMutableStateOfAllProjects(DefaultProjectStateRegistry.java:160)
        at org.gradle.api.internal.project.DefaultProjectStateRegistry.withMutableStateOfAllProjects(DefaultProjectStateRegistry.java:147)
        at org.gradle.execution.DefaultBuildConfigurationActionExecuter.select(DefaultBuildConfigurationActionExecuter.java:37)
        at org.gradle.initialization.DefaultTaskExecutionPreparer.prepareForTaskExecution(DefaultTaskExecutionPreparer.java:39)
        at org.gradle.initialization.VintageBuildModelController.prepareTaskExecution(VintageBuildModelController.java:111)
        at org.gradle.initialization.VintageBuildModelController.doBuildStages(VintageBuildModelController.java:85)
        at org.gradle.initialization.VintageBuildModelController.scheduleRequestedTasks(VintageBuildModelController.java:69)
        at org.gradle.internal.build.DefaultBuildLifecycleController.lambda$scheduleRequestedTasks$1(DefaultBuildLifecycleController.java:114)
        at org.gradle.internal.build.DefaultBuildWorkPreparer.populateWorkGraph(DefaultBuildWorkPreparer.java:28)
        at org.gradle.internal.build.BuildOperationFiringBuildWorkPreparer$PopulateWorkGraph.populateTaskGraph(BuildOperationFiringBuildWorkPreparer.java:120)
        at org.gradle.internal.build.BuildOperationFiringBuildWorkPreparer$PopulateWorkGraph.run(BuildOperationFiringBuildWorkPreparer.java:76)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:29)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:26)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:75)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:68)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:153)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:68)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:56)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.lambda$run$1(DefaultBuildOperationExecutor.java:74)
        at org.gradle.internal.operations.UnmanagedBuildOperationWrapper.runWithUnmanagedSupport(UnmanagedBuildOperationWrapper.java:45)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:74)
        at org.gradle.internal.build.BuildOperationFiringBuildWorkPreparer.populateWorkGraph(BuildOperationFiringBuildWorkPreparer.java:60)
        at org.gradle.internal.build.DefaultBuildLifecycleController.lambda$scheduleRequestedTasks$2(DefaultBuildLifecycleController.java:114)
        at org.gradle.internal.build.DefaultBuildLifecycleController.lambda$withModelOrThrow$5(DefaultBuildLifecycleController.java:142)
        at org.gradle.internal.build.DefaultBuildLifecycleController.withModel(DefaultBuildLifecycleController.java:157)
        at org.gradle.internal.build.DefaultBuildLifecycleController.withModelOrThrow(DefaultBuildLifecycleController.java:140)
        at org.gradle.internal.build.DefaultBuildLifecycleController.scheduleRequestedTasks(DefaultBuildLifecycleController.java:111)
        at org.gradle.internal.buildtree.DefaultBuildTreeWorkPreparer.lambda$scheduleRequestedTasks$0(DefaultBuildTreeWorkPreparer.java:35)
        at org.gradle.composite.internal.DefaultIncludedBuildTaskGraph$1.run(DefaultIncludedBuildTaskGraph.java:113)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:29)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:26)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:75)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:68)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:153)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:68)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:56)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.lambda$run$1(DefaultBuildOperationExecutor.java:74)
        at org.gradle.internal.operations.UnmanagedBuildOperationWrapper.runWithUnmanagedSupport(UnmanagedBuildOperationWrapper.java:45)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:74)
        at org.gradle.composite.internal.DefaultIncludedBuildTaskGraph.lambda$prepareTaskGraph$0(DefaultIncludedBuildTaskGraph.java:110)
        at org.gradle.composite.internal.DefaultIncludedBuildTaskGraph.withState(DefaultIncludedBuildTaskGraph.java:236)
        at org.gradle.composite.internal.DefaultIncludedBuildTaskGraph.prepareTaskGraph(DefaultIncludedBuildTaskGraph.java:107)
        at org.gradle.internal.buildtree.DefaultBuildTreeWorkPreparer.scheduleRequestedTasks(DefaultBuildTreeWorkPreparer.java:34)
        at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.lambda$doScheduleAndRunTasks$1(DefaultBuildTreeLifecycleController.java:83)
        at org.gradle.composite.internal.DefaultIncludedBuildTaskGraph.withNewTaskGraph(DefaultIncludedBuildTaskGraph.java:94)
        at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.doScheduleAndRunTasks(DefaultBuildTreeLifecycleController.java:82)
        at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.runBuild(DefaultBuildTreeLifecycleController.java:104)
        at org.gradle.internal.buildtree.DefaultBuildTreeLifecycleController.scheduleAndRunTasks(DefaultBuildTreeLifecycleController.java:64)
        at org.gradle.tooling.internal.provider.ExecuteBuildActionRunner.run(ExecuteBuildActionRunner.java:31)
        at org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)
        at org.gradle.internal.buildtree.ProblemReportingBuildActionRunner.run(ProblemReportingBuildActionRunner.java:47)
        at org.gradle.launcher.exec.BuildOutcomeReportingBuildActionRunner.run(BuildOutcomeReportingBuildActionRunner.java:69)
        at org.gradle.tooling.internal.provider.FileSystemWatchingBuildActionRunner.run(FileSystemWatchingBuildActionRunner.java:91)
        at org.gradle.launcher.exec.BuildCompletionNotifyingBuildActionRunner.run(BuildCompletionNotifyingBuildActionRunner.java:41)
        at org.gradle.launcher.exec.RootBuildLifecycleBuildActionExecutor.lambda$execute$0(RootBuildLifecycleBuildActionExecutor.java:40)
        at org.gradle.composite.internal.DefaultRootBuildState.run(DefaultRootBuildState.java:154)
        at org.gradle.launcher.exec.RootBuildLifecycleBuildActionExecutor.execute(RootBuildLifecycleBuildActionExecutor.java:40)
        at org.gradle.internal.buildtree.DefaultBuildTreeContext.execute(DefaultBuildTreeContext.java:40)
        at org.gradle.launcher.exec.BuildTreeLifecycleBuildActionExecutor.lambda$execute$0(BuildTreeLifecycleBuildActionExecutor.java:56)
        at org.gradle.internal.buildtree.BuildTreeState.run(BuildTreeState.java:53)
        at org.gradle.launcher.exec.BuildTreeLifecycleBuildActionExecutor.execute(BuildTreeLifecycleBuildActionExecutor.java:56)
        at org.gradle.launcher.exec.RunAsBuildOperationBuildActionExecutor$3.call(RunAsBuildOperationBuildActionExecutor.java:61)
        at org.gradle.launcher.exec.RunAsBuildOperationBuildActionExecutor$3.call(RunAsBuildOperationBuildActionExecutor.java:57)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:200)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:195)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:75)
        at org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:68)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:153)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:68)
        at org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:62)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.lambda$call$2(DefaultBuildOperationExecutor.java:79)
        at org.gradle.internal.operations.UnmanagedBuildOperationWrapper.callWithUnmanagedSupport(UnmanagedBuildOperationWrapper.java:54)
        at org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:79)
        at org.gradle.launcher.exec.RunAsBuildOperationBuildActionExecutor.execute(RunAsBuildOperationBuildActionExecutor.java:57)
        at org.gradle.launcher.exec.RunAsWorkerThreadBuildActionExecutor.lambda$execute$0(RunAsWorkerThreadBuildActionExecutor.java:38)
        at org.gradle.internal.work.DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:211)
        at org.gradle.launcher.exec.RunAsWorkerThreadBuildActionExecutor.execute(RunAsWorkerThreadBuildActionExecutor.java:38)
        at org.gradle.tooling.internal.provider.ContinuousBuildActionExecutor.execute(ContinuousBuildActionExecutor.java:103)
        at org.gradle.tooling.internal.provider.SubscribableBuildActionExecutor.execute(SubscribableBuildActionExecutor.java:64)
        at org.gradle.internal.session.DefaultBuildSessionContext.execute(DefaultBuildSessionContext.java:46)
        at org.gradle.tooling.internal.provider.BuildSessionLifecycleBuildActionExecuter.lambda$execute$0(BuildSessionLifecycleBuildActionExecuter.java:56)
        at org.gradle.internal.session.BuildSessionState.run(BuildSessionState.java:69)
        at org.gradle.tooling.internal.provider.BuildSessionLifecycleBuildActionExecuter.execute(BuildSessionLifecycleBuildActionExecuter.java:55)
        at org.gradle.tooling.internal.provider.BuildSessionLifecycleBuildActionExecuter.execute(BuildSessionLifecycleBuildActionExecuter.java:37)
        at org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:36)
        at org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:25)
        at org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:63)
        at org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:31)
        at org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:58)
        at org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:42)
        at org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:47)
        at org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:31)
        at org.gradle.launcher.daemon.server.exec.ExecuteBuild.doBuild(ExecuteBuild.java:65)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.WatchForDisconnection.execute(WatchForDisconnection.java:39)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.ResetDeprecationLogger.execute(ResetDeprecationLogger.java:29)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.RequestStopIfSingleUsedDaemon.execute(RequestStopIfSingleUsedDaemon.java:35)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.create(ForwardClientInput.java:78)
        at org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.create(ForwardClientInput.java:75)
        at org.gradle.util.internal.Swapper.swap(Swapper.java:38)
        at org.gradle.launcher.daemon.server.exec.ForwardClientInput.execute(ForwardClientInput.java:75)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.LogAndCheckHealth.execute(LogAndCheckHealth.java:55)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.LogToClient.doBuild(LogToClient.java:63)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.EstablishBuildEnvironment.doBuild(EstablishBuildEnvironment.java:84)
        at org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)
        at org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)
        at org.gradle.launcher.daemon.server.exec.StartBuildOrRespondWithBusy$1.run(StartBuildOrRespondWithBusy.java:52)
        at org.gradle.launcher.daemon.server.DaemonStateCoordinator$1.run(DaemonStateCoordinator.java:297)
        at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)
        at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)
        at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:61)


* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

See https://docs.gradle.org/7.2/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 884ms
17 actionable tasks: 1 executed, 16 up-to-date
Watched directory hierarchies: [/Users/donbeave/Projects/myorg/my-project]

```

#### Please tell us about your environment:

| Allure version | 2.14.0 |
| --- | --- |
| Test framework | junit5 |
| Generate report using | allure-gradle@2.9.1 |

It can be related to this issue as well: https://github.com/allure-framework/allure-gradle/issues/60

**Comment by vlsi** [comment]
This is fixed in #74

**Comment by donbeave** [comment]
@vlsi Thank you! It works now.
